lib/components/user/monitored-trip/trip-basics-pane.tsx

```diff
@@ -200,12 +200,11 @@ class TripBasicsPane extends Component<TripBasicsProps> {
                   : monitoredTrip[day]
                   ? 'bg-primary'
                   : ''
-                const notAvailableText = isDayDisabled
-                  ? intl.formatMessage(
-                      { id: 'components.TripBasicsPane.tripNotAvailableOnDay' },
-                      { repeatedDay: getFormattedDayOfWeekPlural(day, intl) }
-                    )
-                  : null
+                const notAvailableText = intl.formatMessage(
```

Comment:
What was the bug that you observed here? It looks like we are currently showing a popup that says "Trip not available on <day>" when the user hovers on a day for which a trip is available, but this change does not seem to fix this particular issue.

Reply:
The bug here was that only Saturday and Sunday was showing up. I think that might be related to the bug you describe, but I think it's more serious to not have the days show then to not have them show in the tooltip

Reply:
Okay, I see. Can you try the original code on this block and remove the `notAvailableText &&` condition in the `return` statement below? I think that's where things broke originally. In `dev` it reads that if a trip **is** available on a particular day, `notAvailable` would be `null` and the label/checkbox for the day is thus not shown.

Reply:
<img width="798" alt="Screen Shot 2022-04-19 at 10 38 49 AM" src="https://user-images.githubusercontent.com/86619099/164029674-39122932-c217-4876-9524-3ddfda354cff.png">

Switching to what you describe causes this to happen for all trips. Not sure if there's more refactoring you want to do here or if you'd rather move this debugging to a different PR

Reply:
I'm not seeing the blanket unavailability.

Reply:
Again, remove the condition on original line 210, and revert original lines 203-208.

Reply:
Resolved in ec4dd4ab